packages/core/src/internal/reconciliation/futures/reconcileReadEventArgument.ts

```diff
@@ -48,8 +48,8 @@ export function reconcileReadEventArgument(
   result = compare(
     future,
     "Argument name",
```

Comment:
```suggestion
    "Argument name or index",
```

Reply:
I guess we should add an equivalent reconciliation rule to static call

Reply:
Done and done